crick/tests/test_stats.py

```diff
@@ -1,4 +1,4 @@
-from __future__ import print_function, division, absolute_import
+from __future__ import absolute_import, division, print_function
```

Comment:
No longer needed with the current slate of supported Python versions

```suggestion
```

Reply:
thanks for catching all of these.